test/verify/check-machines

```diff
@@ -2019,7 +2015,13 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
 
             # For the mocked fedora URL installation wait to see that the OS section is automatically filed
             if self.sourceType == "url" and self.location == TestMachines.TestCreateConfig.TREE_URL:
-                b.wait_attr("label[for=os-select] + div input", "value", TestMachines.TestCreateConfig.FEDORA_28)
+                # virtInstallVersion >= "2" is needed for OS autodetection
```

Comment:
But then this comment doesn't make sense -- all Debian/Ubuntu versions have virt-manager > 2 (see above), so the reason for teh ubuntu/debian special-casing must be something else.